global.h

```diff
@@ -94,6 +94,39 @@ enum langid_t {
 
 #define LXRC_WAIT util_wait(__FILE__, __LINE__, __FUNCTION__);
 
+// LOG_* are bitmasks
+#define LOG_LEVEL_SHOW	(1 << 0)
+#define LOG_LEVEL_INFO	(1 << 1)
+#define LOG_LEVEL_DEBUG	(1 << 2)
```

Comment:
looks likr you would like to have indented values and this line break it ( at least when viewing on github )